frameworks/keyed/ember/src/components/the-table.gjs

```diff
@@ -0,0 +1,27 @@
+import Component from '@glimmer/component';
+import { fn } from '@ember/helper';
+import { getOwner } from '@ember/owner';
+
+export class TheTable extends Component {
+  state = getOwner(this).lookup('service:state');
+
+  <template>
+    <table class="table table-hover table-striped test-data">
+      <tbody>
+        {{#each this.state.data as |row|}}
+          <tr class={{if (this.state.isSelected row) "danger" ""}}><td
```

Comment:
seems we don't need empty class if not provided:
<img width="424" alt="image" src="https://github.com/user-attachments/assets/ba9f8240-ffb7-4342-90ad-59922edcad35" />

Reply:
opened: https://github.com/glimmerjs/glimmer-vm/issues/1716